vcs/vcsvtk/pipeline1d.py

```diff
@@ -103,7 +103,7 @@ def plot(self, data1, data2, tmpl, grid, transform, **kargs):
                 self._context().canvas.plot(m, donotstoredisplay=True)
 
         ren2 = self._context().createRenderer()
-        ren2.SetLayer(1)
+        self._context().setLayer(ren2, tmpl.data.priority)
```

Comment:
@aashish24 Good question. 😄 I don't fully understand the logic there. I think a good time to revisit this is when we implement the multiplot feature.